### DISCUSSION TITLE
AGGrid theme change

### DISCUSSION THREAD

**Discussion by ruidiasbraga** [zauberzeug/nicegui#3111] [Q&A] [answered]
### Question

Hi,

I am trying to change the theme, although I see the minified CSS loaded in the browser I cannot get the grid to change to the desired theme. I am missing out something but cannot figure what. Can someone kindly help? :)

```py
ui.aggrid({
    'defaultColDef': {'flex': 1},
    'columnDefs': [
        {'headerName': 'Content', 'field': 'Content',
         'filter': 'agTextColumnFilter', 'floatingFilter': True},
        {'headerName': 'Content', 'field': 'Content',
         'filter': 'agTextColumnFilter', 'floatingFilter': True},
        {'headerName': 'Content', 'field': 'Content',
         'filter': 'agTextColumnFilter', 'floatingFilter': True},
    ],
    'rowData': rows
}, theme="ag-theme-material")
```

**Comment by ruidiasbraga** [comment]
`}, theme="ag-theme-material").classes('ag-theme-material')`

for my future self and to others, this is the trick :D ...

**Comment by falkoschindler** [comment, marked as answer]
Hi @ruidiasbraga,

The correct solution would be to set `theme` to "material". The "ag-theme-" prefix is added automatically:

```py
ui.aggrid({
    'columnDefs': [
        {'headerName': 'Name', 'field': 'name'},
        {'headerName': 'Age', 'field': 'age'},
    ],
    'rowData': [
        {'name': 'Alice', 'age': 18},
        {'name': 'Bob', 'age': 21},
        {'name': 'Carol', 'age': 42},
    ],
}, theme='material')
```

![Screenshot 2024-05-21 at 18 24 54](https://github.com/zauberzeug/nicegui/assets/5767091/62d83f2b-bec0-4d27-a849-36c0d3e36b2a)
